index.html

```diff
@@ -3373,6 +3376,117 @@ <h2>Definition of Roles</h2>
           </tr>
         </tbody>
       </table>
+    </div>
+    <div class="role" id="label">
+      <rdef>label</rdef>
+      <div class="role-description">
+        <p>A visible name or caption for a user interface component.</p>
+        <p>An element with role <code>label</code> can provide an accessible name for a user interface component if it is programatically associated with the element. Authors MAY associate an element role <code>label</code> with another element by using one of two methods:</p>
```

Comment:
Sentence:
> Authors MAY associate an element role label with another element by using one of two methods:

is missing the word 'with':
> Authors MAY associate an element **_with_** role label with another element by using one of two methods:

Reply:
Thanks for the edit, I made the change